src/coreclr/src/jit/hwintrinsic.h

```diff
@@ -289,7 +310,162 @@ struct HWIntrinsicInfo
 #ifdef TARGET_XARCH
     static int lookupIval(NamedIntrinsic id)
     {
-        return lookup(id).ival;
+        switch (id)
+        {
+            case NI_SSE_CompareEqual:
+            case NI_SSE_CompareScalarEqual:
+            case NI_SSE2_CompareEqual:
+            case NI_SSE2_CompareScalarEqual:
+            case NI_AVX_CompareEqual:
+            {
```

Comment:
Why to put curly braces around simple return statement? They are not needed here, they do not increase readability and I don't believe we use them in other places in the JIT. cc @dotnet/jit-contrib

Reply:
Habit mostly. It minimizes the diff if inserting additional code and allows the block to be collapsed in an IDE.